drafts/2020-05-05-this-week-in-rust.md

```diff
@@ -18,6 +18,23 @@ If you find any errors in this week's issue, [please submit a PR](https://github
 
 * [Multi-threaded HTTP/WebSocket server in Rust](https://sergey-melnychuk.github.io/2020/04/27/multi-threaded-http-websocket-server-in-rust/).
 * [Cheating Rank-n with Traits](https://leshow.github.io/post/cheat_rank_n/)
+* [Understanding Memory and Thread Safety Practices
+and Issues in Real-World Rust Programs](https://cseweb.ucsd.edu/~yiying/RustStudy-PLDI20.pdf)
+* [A beginners guide to gRPC with Rust](https://dev.to/anshulgoyal15/a-beginners-guide-to-grpc-with-rust-3c7o)
+* [Setup Anaconda, Jupyter, and Rust](https://shahinrostami.com/posts/programming/rust-notebooks/setup-anaconda-jupyter-and-rust/)
+* [Teleforking a process onto a different computer!](https://thume.ca/2020/04/18/telefork-forking-a-process-onto-a-different-computer/)
+* [Rust at FullStory, Part 2: A Look Inside Our Mobile SDK](https://bionic.fullstory.com/rust-at-fullstory-part-2/)
+* [Stubbing out WASI manually in Rust](http://www.jakubkonka.com/2020/04/28/rust-wasi-from-scratch.html)
+* [Recon the Hydra Cave: a development note of Rust on GCC](https://nalaginrut.com/archives/2020/04/28/recon%20the%20hydra%20cave%3a%20a%20development%20note%20of%20rust%20on%20gcc%20%28part%201%29)
+* [Educational Rust live coding - Building a git CLI](https://www.youtube.com/watch?v=YFzF1AHYjes)
```

Comment:
Having something like
`* [video] [ Link text ](url)` will help to distinguish videos and blog posts.
Can you do this changes ?

Reply:
Agreed - I'll get this codified somewhere that videos should be labeled like this. This one has been included in a previously merged PR, do you mind deleting this one?

Reply:
Removed the URL